### PR TITLE
WIP: Execute single files with python.

### LINKED ISSUE #5: Create a Class for Execute Python with Java
This need to do:
- [x] Compile a Python file
- [x] Run the compiled file

### PR DESCRIPTION
**Pull request by luqp**

### CODE REVIEW

File: src/main/java/org/jalasoft/moi/model/python/PythonExecutor.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        try {
+            Path pathRun = Paths.get(path.getRoot() + "__pycache__/" + fileName + ".cpython-37.pyc");
+            Process process = Runtime.getRuntime().exec("python " + pathRun);
+            InputStream inputStream = process.getInputStream();
+            InputStreamReader input = new InputStreamReader(inputStream);
+            BufferedReader br = new BufferedReader(input);
+            String line = null;
+            System.out.println("<OUTPUT>");
+            while ((line = br.readLine()) != null)
+                System.out.println(line);
```
Comment (nanrehd) [inline src/main/java/org/jalasoft/moi/model/python/PythonExecutor.java]: This while needs its braces as it is not empty.

**Review by nanrehd — COMMENTED** [review]
Please don't forget to comment a little about your code, and put the copyright :)

File: src/main/java/org/jalasoft/moi/model/python/PythonExecutor.java (hunk 1 above)
Comment (luqp) [reply src/main/java/org/jalasoft/moi/model/python/PythonExecutor.java]: Thanks you, I added standards now.

**Review by nanrehd — APPROVED** [review]

**Review by carlosmc23 — APPROVED** [review]

**Comment by carlosmc23** [comment]
Please follow branch conventions for pull request

**Review by Snip-77 — APPROVED** [review]

**Pull request closed without merging**